### PR TITLE
Add type annotations for emoji

### PR DESCRIPTION
**Pull request by ASMfreaK**
This adds type annotations for [`emoji` library](https://github.com/carpedm20/emoji/)

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
Can you ask the maintainer of the library for permission to add stubs on typeshed and link that here?

File: third_party/2and3/emoji.pyi [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from typing import Tuple, Pattern, List, Dict, Union
+
+_DEFAULT_DELIMITER = ':'
+
+
+def emojize(
+    string: str,
+    use_aliases: bool=False,
```
Comment (JelleZijlstra) [inline third_party/2and3/emoji.pyi]: Defaults should always just be `...`.

**Comment by ASMfreaK** [comment]
Ok. I've written the issue carpedm20/emoji#64 and sent an email to author. Hopefully, we'll get a reponce soon enough.

**Comment by carpedm20** [comment]
This looks no problem to me. Thanks for the work @ASMfreaK

**Comment by gvanrossum** [comment]
Thanks!

**Pull request merged**